sdks/python/apache_beam/transforms/sql_test.py

```diff
@@ -157,6 +157,21 @@ def test_zetasql_generate_data(self):
           dialect="zetasql")
       assert_that(out, equal_to([(1, "foo", 3.14)]))
 
+  def test_windowing_before_sql(self):
+    with TestPipeline() as p:
+      windowed = (
+          p | beam.Create([
+              SimpleRow(5, "foo", 1.),
+              SimpleRow(15, "bar", 2.),
+              SimpleRow(25, "baz", 3.)
+          ])
+          | beam.Map(lambda v: beam.window.TimestampedValue(v, v.id)).
+          with_output_types(SimpleRow)
+          | beam.WindowInto(
+              beam.window.FixedWindows(10)).with_output_types(SimpleRow)
+          | SqlTransform("SELECT COUNT(*) as `count` FROM PCOLLECTION"))
+      assert_that(out, equal_to([(1, ), (1, ), (1, )]))
```

Comment:
Just curios, Is this failing before the fix or does it pass with a wrong pipeline ?

Reply:
Before the fix it replicates the failure here: https://issues.apache.org/jira/browse/BEAM-10143

It fails at execution time because it expects a global window, but receives a (correct) interval window